### ISSUE TITLE
Change Misleading Comparison Table

### ISSUE THREAD

**Issue by presentfactory** [simplex-chat/website#34]
While what SimpleX tries to do seems to be good I am continuously bothered by the comparison table put prominently on the site as it feels like it is deceptively trying to set SimpleX above peer to peer protocols despite many of them being pretty close to what is being offered.

**Possibility of MITM**

Saying Signal, Matrix, or P2P protocols have the "possibility" of MITM while SimpleX is not is kinda nonsense. The footnote says SimpleX prevents this with out of band key exchange/verification, but that's true with any online communication. It is /always/ possible to MITM communications through the internet, it does not matter what protocol it is. The only way to combat it is to do the key exchange via some trusted channel (e.g. out of band like SimpleX and P2P protocols like Tox do) or verify it via a trusted channel that the key exchange was done properly (as is done with Matrix and others do with a hash).

As such I'd personally just remove this row as any good e2e encrypted secure messaging application should be resistant to MITM attacks (as that's kinda the whole point), so it's not really providing any meaningful comparison. If you want to keep it, a new column should be added to compare to things that offer e2e encryption but don't actually give you ways to do the key exchange yourself and/or verify that it was done properly. Examples of that are things that work on more of a "Trust me bro" principle for the key exchange such as Apple's FaceTime, or any proprietary software as you cannot inspect the code easily to ensure the crypto is actually doing as it claims in such cases.

**Single or Centralized network**

This whole row is also fairly confusing as it conflates single and centralized networks together when really they are only a bad thing when combined. Centralization we know is bad as it leads often to a single network you must use which has control over everything, but this does not mean having a single network is a bad thing. For instance in P2P sure with many implementations there's only a single network, but unlike a centralized single network there is no central actual authority over it due to its distributed nature, so it already avoids the problem centralization has.

Additionally, federated networks often form a singular network just the same like P2P networks do, that's kinda the whole point of federation (allowing them to communicate as part of one big network rather than just a number of scattered centralized servers). Sure you can in theory form a new distinct federated network if you really tried to, but you can do the same with P2P too. Nothing about either technology enforces a single network other than that usually being the default configuration to not have the entire userbase fragmented across 20 networks, and this is desired because a single network in such decentralized systems avoids the authority concerns that arise when centralized services do this (that's kinda the whole point of structuring the protocol in this way).

So for this one I'd change the P2P one to "No - distributed" because if federated protocols are going to be considered fine in this regard then P2P should as well (as P2P is the extreme case of any sort of decentralized protocol). I'd also remove the "single network" bit and change it to just "Centralized network" as that is not the problem. The "problem" is centralized authority things like Signal have as it gives a single entity control over things with no alternatives unlike the lack of authority in P2P networks or the ability to create alternative federated instances in federated networks.

**Central component or other network-wide attack**

For this one I really don't understand what it's getting at and once again seems a bit useless of a comparison. Yes there's attacks against every network architecture out there, really a denial of service attack can be applied to any such thing. Usually P2P networks and the like are considered more robust against DDoS in some ways due to how many nodes have to be targeted to actually hurt the network significantly, but of course Sybil attacks and other such things with bad nodes can cause their own sorts of problems. Similarly the lack of central networking can make it harder to mitigate such attacks, vs a big company simply using a DDoS protection service to absorb the traffic and blacklist specific IP ranges across the entire service etc.

All that said, I do not how SimpleX thinks it is "resilient" enough to be classified as a "No" in this category whereas all these other networks are not. SimpleX is not magically immune to attacks from what I understand, one can simply DDoS the message queues and everything will fall apart much like one can do with a federated network's servers.

Finally, nowhere is the claim "P2P networks either have a central authority or the whole network can be compromised" actually explained. If it is referring to Sybil attacks then sure but that's just a denial of service attack and the same can be said about anything else. Is it easier to do than other network types? Maybe in some ways with that sort of approach, but it's not like P2P networks are totally oblivious to the concern either (hence stuff like cryptocurrencies and proof of work/stake systems to mitigate such things due to how important the validity of the consensus is in those cases).

Tldr, a lot of the stuff on the site seems to be inaccurate or misleading to try and make SimpleX look significantly better than stuff like P2P networks which offer many of the same strengths. Hopefully someone can fix this.

**Comment by presentfactory** [comment]
Actually I see this repo is supposed to be archived, probably should've checked that before. I guess I'll move this complaint over to the main simplex-chat repo.
